templates/skeleton/server.ts

```diff
@@ -33,6 +33,20 @@ export default {
         throw new Error('SESSION_SECRET environment variable is not set');
       }
 
+      const {pathname} = new URL(request.url);
+      const wpmAssets = pathname.startsWith('/wpm@');
+      if (wpmAssets) {
+        console.log();
+        const wpmUrl = `https://${env.PUBLIC_STORE_DOMAIN}/${pathname}`;
+        const wpmResponse = await fetch(wpmUrl);
+
+        // Doesn't work because CF doesn't allow cross origin html requests
```

Comment:
But this `fetch` is going from the server, right? So there shouldn't be cross-origin?
Or you mean when returning this response to the browser? In that case we can modify headers here to remove the cross-origin related stuff before returning the response 🤔